pkg/collector/corechecks/servicediscovery/usm/php_test.go

```diff
@@ -50,7 +50,8 @@ func TestServiceNameFromCLI(t *testing.T) {
 			expected: "",
 		},
 	}
-	instance := &phpDetector{ctx: NewDetectionContext(nil, envs.NewVariables(nil), nil)}
+	dc := NewDetectionContext(0, nil, envs.NewVariables(nil), nil, nil)
```

Comment:
Maybe if you leave the signature of NewDetectionContext() as-is but add a new constructor for the callsite which need it then you don't have to go in and another nil everywhere?

Reply:
I've restored signature of NewDetectionContext()